src/averageAges.js

```diff
@@ -15,11 +15,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const men = people.filter(person => person.sex === 'm');
+  const mappedMen = century
+    ? men.filter(person =>
+      Math.ceil(person.died / 100) === century).map(person =>
+      person.died - person.born)
+    : men.map(person => person.died - person.born);
```

Comment:
```suggestion
  const mappedMen = people.filter(person => century 
    ? Math.ceil(person.died / 100) === century) && person.sex === 'm'
    : person.sex === 'm';
```
По аналогии сделай в остальных функциях